### PR TITLE
Add org.freedesktop.Sdk.Extension.php83

### PR DESCRIPTION
**Pull request by Lctrs**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by Lctrs** [comment]
bot, build org.freedesktop.Sdk.Extension.php83

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "automerge-flathubbot-prs": true,
```
Comment (bbhtt) [inline flathub.json:2]: Don't automerge PRs, they'll get merged without any oversight

Comment (Lctrs) [reply flathub.json:2]: That's what has already been done for the previous php extensions.

As the maintainer, I'm OK with it.

**Comment by Lctrs** [comment]
bot, build org.freedesktop.Sdk.Extension.php83

File: flathub.json (hunk 1 above)
Comment (Lctrs) [reply flathub.json]: Okay, I've removed it.

**Pull request closed without merging**